examples/OcclusionSaliency.ipynb

```diff
@@ -1,151 +1,301 @@
 {
  "cells": [
   {
-   "cell_type": "code",
-   "execution_count": 1,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "\"\"\"\n",
-    "This notebook contains a working example show usage of the new\n",
-    "image classifier based saliency map API for choosing to use \n",
-    "a occlusion based saliency map algorithm.\n",
+    "# Introduction\n",
+    "This notebook contains a working example to show usage of the image perturbation and classifier-based saliency map generation API.\n",
+    "\n",
+    "This example will follow an application-like use-case where we define a functionally rigid process that transforms an input image into a number of saliency heat-maps based on our black-box classifier's output, visualizing them over the input image.\n",
+    "We will show that it is easy to change which of our API implementations are used in the application without impacting the application's successful execution, first using a sliding-window perterbation algorithm and then using the RISE perturbation algorithm.\n",
+    "\n",
+    "This will necessarilly include the use of a classification model to perform the role of the black-box classifier that we will be determining the saliency for.\n",
+    "We will fill this role here with a PyTorch Imagenet-pretrained ResNet18 network.\n",
     "\n",
-    "Packages required for the below example\n",
+    "### Table of Contents\n",
+    "* [The test image](#The-test-image)\n",
+    "* [The \"application](#The-\"application\")\n",
```

Comment:
Missing end quotes:

```
[The \"application\"]
```
The links in the TOC don't render for me in Github, but maybe they do in an actual notebook.

Reply:
I want to say that is a Github issue as this is clearly rendering and working in my jupyter-notebook instance. Below are my installed jupyter versions if anything looks off / non-standard:
```bash
$ jupyter --version
jupyter core     : 4.7.1
jupyter-notebook : 6.4.0
qtconsole        : 5.1.0
ipython          : 7.23.1
ipykernel        : 5.5.5
jupyter client   : 6.1.12
jupyter lab      : not installed
nbconvert        : 6.0.7
ipywidgets       : 7.6.3
nbformat         : 5.1.3
traitlets        : 5.0.5
```